### ISSUE TITLE
[Help] [Docs request] No clear indication of how to use the "Methods"

### ISSUE THREAD

**Issue by BeamZack** [DethAriel/ng-recaptcha#80]
### Summary
I want to use the `reset()` method but its not clear the best way to do this, or how to at all.

I'm submitting a: 
- [ ] bug report
- [ ] feature request
- [X] question / support request
- [ ] other

### Description
*Disclaimer*
I only upgraded to angular 4 a month ago so I'm not sure if I am missing something key to the framework that would be obvious to me with more experience.  If I am, I apologies and if you could link me to the Angular docs where it is covered I would be grateful. 

My problem is simple:
I have a `<re-captcha>` in my view for a signup page.  I have error handling on blur of the fields, but I also leave the button enabled and have data validation from 3rd party services.  So if the user enters incorrect data I need to reset the captcha as I don't want to reload the entire page.

Enter `reset()`.  This is exactly what I want to do. But where do I call it? I have no captcha object in my component.ts file.  My setup is very similar to the `Basic Usage` example but there is nothing in the documentation on how Methods would be used, only that they exist. 

tl;dr `______.reset();` what is that blank supposed to be?

**Comment by BeamZack** [comment]
Interesting.  I added `grecaptcha.reset();` to my submit function and it worked.  I'm not sure what magic is happening. `grecaptcha` seems to be the global that the underlying package uses and so it is a global here as well.

This feels like I'm doing it the wrong way.

If this is correct, my problem is solved but I still think the readme would benefit from small change to lay this out.

**Comment by DethAriel** [comment]
@BeamZack, accessing these methods is no different from accessing any other components methods in Angular, and can be done in various ways, for example:

```
<re-captcha #captchaRef="reCaptcha"></re-captcha>
<button (click)="captchaRef.reset()">Reset</button>
```

You can see that as an example [here](https://dethariel.github.io/ng-recaptcha/invisible).

Another way of doing this is described on [Angular documentation website](https://angular.io/guide/component-interaction#parent-calls-an-viewchild).
